waterlog/src/actions/Types.js

```diff
@@ -96,6 +99,12 @@ export const FETCH_COSTS_FORECAST_FAILURE_MONTHLY = 'FETCH_COSTS_FORECAST_FAILUR
 export const FETCH_HEAT_MAP_DATA_BEGIN = 'FETCH_HEAT_MAP_DATA_BEGIN';
 export const FETCH_HEAT_MAP_DATA_SUCCESS = 'FETCH_HEAT_MAP_DATA_SUCCESS';
 export const FETCH_HEAT_MAP_DATA_FAILURE = 'FETCH_HEAT_MAP_DATA_FAILURE';
+//redux saga actions
+export const FETCH_NOTIF = 'FETCH_NOTIF';
```

Comment:
Plan for your next sprint. Group these into a meaningful constant object so you guys don't have to individually import and export each line. 
Eg combine all map data in to one or all heat maps into one

Reply:
Thanks .